js/exporter/jspdf/export_data_grid_2.js

```diff
@@ -26,6 +26,12 @@ function exportDataGrid(doc, dataGrid, options) {
                     if(options.onCellExporting) {
                         options.onCellExporting({ gridCell: { value: cellData.value }, pdfCell });
                     }
+                    if(isDefined(options.columnWidths)) {
+                        const width = options.columnWidths[cellIndex];
+                        if(width && !pdfCell.skip && !pdfCell.rect.w) {
```

Comment:
'pdfCell.rect.w' can be '0'
let's use '!isDefine(pdfCell.rect.w)'

Reply:
And 'if(width' - can be '0'

Reply:
Whether we need a new test?

Reply:
Made changes to the code and added a test for zero column width